live-examples/js-examples/weakset-prototype-add.html

```diff
@@ -1,6 +1,6 @@
 <pre>
-<code id="static-js">var weakset1 = new WeakSet();
-var object1 = {};
+<code id="static-js">const weakset1 = new WeakSet();
+const object1 = {};
```

Comment:
Very nitpicky, but in Firefox the output is `TypeError: WeakSet value must be an object, got the number 1`

(that is, TypeError not Error)